### ISSUE TITLE
Install with embeddings

### ISSUE THREAD

**Issue by marcglobality** [pytorch/text#1069]
I would like to install torchtext with glove embeddings, so that when I deploy the Docker image it doesn't have to download them every time. Is there any good way (other than do a hacky wget myself and place them in the correct directory) to do this from CLI?

e.g. 

`pip install torchtext[glove]`

or as in spacy

`python -m torchtext download glove`

Thanks!

**Comment by zhangguanheng66** [comment]
Definitely. If you switch to the new glove vectors ([link](https://github.com/pytorch/text/blob/97e6d1d64b82ff91899b9083350951d8619e5914/torchtext/experimental/vectors.py#L63)), the glove files will be downloaded once when you first use it. But keep in mind, the new glove vectors is in the nightly release so you install the nightly package
```
pip install --pre torch torchtext -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```
We also provide pip and conda packages for Windows/MacOS/Linux.

**Comment by marcglobality** [comment]
Hi @zhangguanheng66 , thanks for your quick answer. What I would like is to have some pip command to download them **before using them**, so that they are saved in ~. 

This way, every time I start the docker image in a different box, it doesn't have to download them the first time they are used

**Comment by zhangguanheng66** [comment]
OK, then this is a new feature request.

**Comment by marcglobality** [comment]
> OK, then this is a new feature request.

by your answer, I understand this is not possible right now (?)

**Comment by zhangguanheng66** [comment]
> > OK, then this is a new feature request.
> 
> by your answer, I understand this is not possible right now (?)

That's right. A temporary solution is to download the glove embedding with shell script.
